AutomationCalculator.js jsx-curly-spacing linting errors

**Describe the bug**

Running the frontend with a newer npm version (6.14.5) reveals a linting error in the Automation Calculator component.

```
ERROR in ./src/Containers/AutomationCalculator/AutomationCalculator.js                                                                             
Module Error (from ./node_modules/eslint-loader/index.js):                                                                                                                                                                                                                                            
                                                                         
/app/src/Containers/AutomationCalculator/AutomationCalculator.js                                                                                                                                                                                                                                      
  363:7  error  A space is required after '{'   react/jsx-curly-spacing                                                                            
  371:8  error  A space is required before '}'  react/jsx-curly-spacing                                                                                                                                                                                                                               
  372:7  error  A space is required after '{'   react/jsx-curly-spacing
  621:8  error  A space is required before '}'  react/jsx-curly-spacing                                                                                                                                                                                                                               
                                                                                                                                                   
✖ 4 problems (4 errors, 0 warnings)                                                                                                                                                                                                                                                                   
  4 errors and 0 warnings potentially fixable with the `--fix` option.
```                                                                                

**To Reproduce**
1. npm install -g npm@6.14.5
2. npm install
3. npm run start

**Expected behavior**
npm run succeeds

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: fedora 32
 - Browser N/A
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/src/Containers/AutomationCalculator/AutomationCalculator.js
+++ b/src/Containers/AutomationCalculator/AutomationCalculator.js
@@ -360,7 +360,7 @@ const AutomationCalculator = () => {
       <PageHeader style={ { flex: '0' } }>
           <PageHeaderTitle title={ title } />
       </PageHeader>
-      {preflightError && (
+      { preflightError && (
           <Main>
               <Card>
                   <CardBody>
@@ -368,8 +368,8 @@ const AutomationCalculator = () => {
                   </CardBody>
               </Card>
           </Main>
-      )}
-      {!preflightError && (
+      ) }
+      { !preflightError && (
         <>
           <Wrapper className="automation-wrapper">
               <WrapperLeft>
@@ -618,7 +618,7 @@ const AutomationCalculator = () => {
               </WrapperRight>
           </Wrapper>
         </>
-      )}
+      ) }
     </>
     );
 };

